### PR TITLE
Add bitcoin-core-0.21.0-new-release to bitcoin-magazine/bitcoin-explained

### PR DESCRIPTION
**Pull request by Bitc0indad**
This PR adds [Bitcoin Core 0.21.0 - New Release](https://www.youtube.com/watch?v=kM4912dv39o) transcript to the bitcoin-magazine/bitcoin-explained directory.

### CODE REVIEW

**Comment by Bitc0indad** [comment]
First time submission please reach out with feedback. Looking forward to contributing more.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md [hunk 1]
```diff
@@ -0,0 +1,915 @@
...
+Yeah, it makes it very obvious which addresses, which inputs originate from which node, which is bad for privacy, because a node can be tied to an IP address and an IP address to an identity and so forth.
+
+Sjors Provost: 00:04:03
+
+Exactly. So the long-term goal here, which has not been achieved in this specific change, but we always do little micro steps.
+
+Aaron van Wirdum: 00:04:11
+
+Uh huh
+
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md:154]: This kind of lines don't contribute to the transcript's content - you can just delete this and continue with the same speaker. This transcript has a lot back n forth between participants, so you'll notice the presence of numerous filler words or non-essential lines that need removal.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md [hunk 2]
```diff
@@ -0,0 +1,915 @@
...
+Exactly and this also creates a better separation between because then you know Bitcoin Core is a node which has a mempool and it's a wallet and you know this dealing with mempools should be done by the node, and it shouldn't be done by the wallet. So there's a lot of process separation going on under the hood there.
+
+Aaron van Wirdum: 00:05:25
+
+Yeah, but this, what we're describing right now, is actually not included in Bitcoin Core 21 right now. We've seen one micro step towards this in Bitcoin Core 21.
+
+Sjors Provost: 00:05:26
+
+Yeah.
+
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md:174]: Another example of a line that doesn't contribute to the transcript's content - you can just delete this and continue with the same speaker.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md [hunk 3]
```diff
@@ -0,0 +1,915 @@
...
+Yeah, exactly.
+
+Aaron van Wirdum: 00:09:40
+
+That's why you remember the exact number.
+
+Sjors Provost: 00:09:42
+
+Precisely, it's very hard to remember. There are lots of trade-offs when it comes to these filters, so I think we should go into that for another episode. It's not enforced by consensus, so you're kind of trusting that the list is real. Although if you...Let's say the list gives you a bunch of nonsense results and you start downloading those blocks and you don't see what you're expecting, well then you know that whoever gave you those filters was lying to you.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md:339]: Here "Although if you..." is an example of mid-sentence rephrasing - eliminating it helps with readability.

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md [hunk 4]
```diff
@@ -0,0 +1,915 @@
...
+Right, yeah.
+
+Sjors Provost: 00:14:50
+
+So, even if the rules change now, there's no problem with your 0.21 node.It will not get confused if Taproot changes. Of course, hopefully it doesn't.
+
+Aaron van Wirdum: 00:14:59
+
+Sure, But about that, so yeah, the taproot code is included in Bitcoin Core 21.
+This also means that probably the next minor version of Bitcoin Core, which would be Bitcoin Core 21.1, well, Bitcoin Core 0.21.1, will have the actual activation logic for TapRoot.
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md:473]: "[taproot](https://bitcoinops.org/en/topics/taproot/)" is the correct way of writing it, not "TapRoot".

File: bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md [hunk 5]
```diff
@@ -0,0 +1,915 @@
...
+Aaron van Wirdum: 00:16:15
+
+Yeah there's more on that discussion indeed in episode 3 and taproot itself in episode 2 all right next point
+
+## BIP339 "WTXID"
+
+Sjors Provost: 00:16:22
+
+yeah so we'll jump back in our little queue to BIP339 witness transaction id relay or WTXID so here's the thing a long long time ago before there was SegWit, transactions had a hash.
+And this hash represented, you know, the entire transaction. But...
```
Comment (kouloumos) [inline bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md:513]: "But..." is another example of a non-essential line that can be deleted.

**Review by kouloumos — COMMENTED** [review]
Thank you for your contribution @Bitc0indad! Great work in identifying AI's transcription errors and ensuring sentence coherence, especially in instances where they were disrupted by chapter titles or misattributed words from one speaker to another.

I've left a few comments inline with some examples of non-essential lines that you can remove. As we continuously strive to enhance productivity, we'll soon provide more comprehensive Review Guidelines to clarify expectations. In the meantime, please read the Review Guidelines in https://github.com/bitcointranscripts/bitcointranscripts/issues/348, and you can see an examples of 'right editing' in https://github.com/bitcointranscripts/bitcointranscripts/issues/335.

Also, for future reviews, have in mind that you should maintain the original "one-sentence-per-line" formatting.

Check the links above, and we can merge this as soon as you update the review based on the provided guidelines. Thank you!

**Comment by Bitc0indad** [comment]
@kouloumos thank you for the quick feedback, I will push an updated markdown file shortly.

**Comment by Bitc0indad** [comment]
@kouloumos updated. Additional feedback is greatly appreciated! If all set, thanks again!

**Comment by kouloumos** [comment]
Thank you for the work you've put into your review @Bitc0indad. The extra chapters and tags you've added really elevate the quality and organization of the content, and your dedication to enhancing the transcript is truly appreciated.

I've gone ahead and made [a few tweaks of my own](https://github.com/bitcointranscripts/bitcointranscripts/pull/387/commits/c762337017695315c689bacee8068e54d1439f9f) that you might want to review before tackling your next submission. These edits are meant to give you a bit more insight into the direction we're aiming for.

One specific piece of feedback I'd like to share is about streamlining the transcript for readability. After removing any non-essential lines, consider merging segments from the same speaker into a single block. This helps maintain a clean and cohesive flow. Additionally, it's important to stick to the original "one-sentence-per-line" formatting.

If you have any questions, or if you're looking for tips on editing, feel free to reach out on our [Discord channel](https://discord.gg/76wCxgeyvM). It's a great space for real-time feedback, connecting with other contributors, and receive real-time notifications for new transcripts available for review.

Thanks again for your contribution! Looking forward to seeing your next review!

**Comment by Bitc0indad** [comment]
much appreciated @kouloumos this is a great way to provide feedback to plebs trying to become devs who can do basic markdown updates and then push the PR for feedback loop review. it is perfect, and I will continue to contribute!

**Pull request merged**